include/openPMD/IO/ADIOS/ADIOS2Auxiliary.hpp

```diff
@@ -36,6 +36,219 @@
 
 namespace openPMD
 {
+/** Concrete datatype of an object available at runtime.
+ * Since ADIOS2 distinguishes unsigned char, signed char and char, this enum
+ * has all three of them (unlike the public Datatype enum)
+ */
+enum class ADIOS2Datatype : int
+{
+    CHAR,
+    UCHAR,
+    SCHAR,
+    SHORT,
+    INT,
+    LONG,
+    LONGLONG,
+    USHORT,
+    UINT,
+    ULONG,
+    ULONGLONG,
+    FLOAT,
+    DOUBLE,
+    LONG_DOUBLE,
+    CFLOAT,
+    CDOUBLE,
+    CLONG_DOUBLE,
+    STRING,
+    VEC_CHAR,
+    VEC_UCHAR,
+    VEC_SCHAR,
+    VEC_SHORT,
+    VEC_INT,
+    VEC_LONG,
+    VEC_LONGLONG,
+    VEC_USHORT,
+    VEC_UINT,
+    VEC_ULONG,
+    VEC_ULONGLONG,
+    VEC_FLOAT,
+    VEC_DOUBLE,
+    VEC_LONG_DOUBLE,
+    VEC_CFLOAT,
+    VEC_CDOUBLE,
+    VEC_CLONG_DOUBLE,
```

Comment:
Maybe remove `VEC_CLONG_DOUBLE`?

Reply:
-> comment out

Reply:
Yes, let's comment this here since ADIOS2 does not support it.